plugins/Dashboard/Updates/4.7.0.php

```diff
@@ -0,0 +1,61 @@
+<?php
+/**
+ * Piwik - free/libre analytics platform
+ *
+ * @link    https://matomo.org
+ * @license http://www.gnu.org/licenses/gpl-3.0.html GPL v3 or later
+ */
+
+namespace Piwik\Plugins\Dashboard;
+
+use Piwik\Updater;
+use Piwik\Updates as PiwikUpdates;
+use Piwik\Updater\Migration;
+use Piwik\Updater\Migration\Factory as MigrationFactory;
+
+/**
+ * Update for version 1.0.8-beta.
+ */
+class Updates_4_7_0 extends PiwikUpdates
```

Comment:
This should have the name of the next possible beta release. Currently that would be `4.7.0-b2` and in order to have the script applied when someone fetches your changes you might also need to change the version in `core\Version.php`